src/Metadata/Resource/Factory/XmlResourceMetadataFactory.php

```diff
@@ -83,9 +83,9 @@ private function getMetadata(string $resourceClass) : array
                 }
 
                 return [
-                    (string) $resource['shortName'] ?? null,
-                    (string) $resource['description'] ?? null,
-                    (string) $resource['iri'] ?? null,
+                    (string) $resource['shortName'] ?: null,
```

Comment:
Correct me if I am wrong but if an attribute does not exist in a `\SimpleXMLElement` class, its value is `null` so the result of this expression is never `null` (i.e. `(string) null` => `''`).

Example with `$resource['foo']` which does not exist:
```php
var_dump($resource); // class SimpleXMLElement
var_dump($resource['foo']); // NULL
var_dump((string) $resource['foo'] ?? null); // string(0) ""
var_dump((string) $resource['foo'] ?: null); // NULL
```